docs/website/versioned_docs/version-3.0.0/command-reference/dev.md

```diff
@@ -226,3 +226,25 @@ components:
     memoryLimit: 1024Mi
     mountSources: true
 ```
+
+### State file
+
+When the command `odo dev` is executed, the state of the command isa saved in the file `./.odo/state.json`. 
+
+This state file contains the PID of the `odo dev` running process (`0` if no command is running), the timestamp 
+at which the state has been last saved, and the currently forwarded ports.
+
+```
+{
+ "timestamp": 1651247240,
+ "pid": 154474,
+ "forwardedPorts": [
+  {
+   "containerName": "runtime",
```

Comment:
Maybe we should update the example in the description of the original issue: the field is named `container` there..

Reply:
done, thanks